.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "dotnet-format": {
+      "version": "5.1.225507",
```

Comment:
@jgiannuzzi per dotnet-format repo, this one is the latest one https://github.com/dotnet/format/releases/tag/v5.1.225507